core/modules/qdisp/Executive.cc

```diff
@@ -556,8 +585,8 @@ void Executive::_waitAllUntilEmpty() {
 }
 
 std::ostream& operator<<(std::ostream& os,
-                         Executive::StatusMap::value_type const& v) {
-    ExecStatus const& es = *(v.second);
+                         Executive::JobStatusPtrMap::value_type const& v) {
+    JobStatus const& es = *(v.second);
     os << v.first << ": " << es;
```

Comment:
no need to create "es", you can write it as os << v.first << ": " << *(v.second);